cmd/juju/application/removeapplication_test.go

```diff
@@ -43,70 +44,88 @@ func (s *RemoveServiceSuite) SetUpTest(c *gc.C) {
 	s.PatchValue(&getBudgetAPIClient, func(*httpbakery.Client) budgetAPIClient { return s.budgetAPIClient })
 }
 
-func runRemoveService(c *gc.C, args ...string) error {
-	_, err := testing.RunCommand(c, NewRemoveServiceCommand(), args...)
+func runRemoveApplication(c *gc.C, args ...string) error {
+	_, err := testing.RunCommand(c, NewRemoveApplicationCommand(), args...)
 	return err
 }
 
-func (s *RemoveServiceSuite) setupTestService(c *gc.C) {
+func (s *RemoveApplicationSuite) setupTestApplication(c *gc.C) {
 	// Destroy an application that exists.
 	ch := testcharms.Repo.CharmArchivePath(s.CharmsPath, "riak")
 	err := runDeploy(c, ch, "riak", "--series", "quantal")
 	c.Assert(err, jc.ErrorIsNil)
 }
 
-func (s *RemoveServiceSuite) TestSuccess(c *gc.C) {
-	s.setupTestService(c)
-	err := runRemoveService(c, "riak")
+func (s *RemoveApplicationSuite) TestLocalApplication(c *gc.C) {
+	s.setupTestApplication(c)
+	err := runRemoveApplication(c, "riak")
 	c.Assert(err, jc.ErrorIsNil)
 	riak, err := s.State.Application("riak")
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(riak.Life(), gc.Equals, state.Dying)
 	s.stub.CheckNoCalls(c)
 }
 
-func (s *RemoveServiceSuite) TestRemoveLocalMetered(c *gc.C) {
+func (s *RemoveApplicationSuite) TestRemoteApplication(c *gc.C) {
```

Comment:
the suite also contains tests for adding remote relations against a remote app ie TestAddRemoteRelation() so the test name is ok I think